cmake/config.cmake

```diff
@@ -104,7 +104,12 @@ set(OMR_TOOLS_USE_NATIVE_ENCODING ON CACHE BOOL
 ### Library names
 ###
 
-set(OMR_GC_LIB "omrgc" CACHE STRING "Name of the GC library to use")
+# Set the OMR_GC_LIB based on pointer mode; in mixed references mode, use compressed omrgc library
+if(OMR_GC_COMPRESSED_POINTERS)
+	set(OMR_GC_LIB "omrgc" CACHE STRING "Name of the GC library to use")
+elseif(OMR_GC_FULL_POINTERS)
+	set(OMR_GC_LIB "omrgc_full" CACHE STRING "Name of the GC library to use")
+endif()
```

Comment:
This is what's causing the build failures. Is there a better way of doing this? When building locally, `touch`ing the file seems to alleviate this problem.

Reply:
I think we need two variables so `gc/api/CMakeLists.txt` can use the right name for each shared library.